courses/management/commands/test_manage_program_certificates.py

```diff
@@ -0,0 +1,60 @@
+"""Tests for Program Certificates management command"""
+
+import pytest
+from courses.management.commands import manage_program_certificates
+from courses.models import ProgramCertificate
+from django.core.management.base import CommandError
+from courses.factories import (
+    CourseFactory,
+    CourseRunFactory,
+    CourseRunGradeFactory,
+    CourseRunCertificateFactory,
+    ProgramFactory,
+    ProgramEnrollmentFactory,
+)
+
+pytestmark = [pytest.mark.django_db]
+
+
+def test_program_certificate_management_no_argument():
+    """Test that command throws error when no input is provided"""
+
+    with pytest.raises(CommandError) as command_error:
+        manage_program_certificates.Command().handle()
+    assert str(command_error.value) == "Please provide a valid program readable_id."
+
+
+def test_program_certificate_management_invalid_program():
+    """
+    Test that program certificate management command throws proper error when
+    no valid program is supplied
+    """
+
+    with pytest.raises(CommandError) as command_error:
+        manage_program_certificates.Command().handle(readable_id="test")
+    assert (
+        str(command_error.value)
+        == "Could not find program enrollment(s) with readable_id=test"
+    )
+
+
+def test_program_certificate_creation(user):
```

Comment:
We should also write a test when there are multiple courses in the program and not all of them have certificates, In that case the command should not create a program certificates for the program